code/modules/projectiles/guns/energy/experimental.dm

```diff
@@ -236,51 +241,262 @@
 		unwield(user)
 	else
 		wield(user)
-/*
+
+/obj/item/ammo_casing_gatling
+	name = "large bullet casing"
+	desc = "An oversized bullet casing."
+	icon = 'icons/obj/ammo.dmi'
+	icon_state = "gatling-casing"
+	flags = FPRINT
+	siemens_coefficient = 1
+	throwforce = 1
+	w_class = 1.0
+	w_type = RECYK_METAL
+
+/obj/item/ammo_casing_gatling/New()
+	..()
+	pixel_x = rand(-10.0, 10)
+	pixel_y = rand(-10.0, 10)
+	dir = pick(cardinal)
+
+
+#define MAX_STICKYBOMBS 4
+
 /obj/item/weapon/gun/stickybomb
 	name = "stickybomb launcher"
-	desc = "Ya-ta-ta-ta-ta-ta-ta-ta ya-ta-ta-ta-ta-ta-ta-ta do-de-da-va-da-da-dada! Kaboom-Kaboom!"
+	desc = "Fired stickybombs take 5 seconds to become live. After which they'll progressively merge with their surroundings."
 	icon = 'icons/obj/gun_experimental.dmi'
-	icon_state = "minigun"
-	item_state = "minigun0"
+	icon_state = "stickybomb"
+	item_state = null
 	origin_tech = null
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guns_experimental.dmi', "right_hand" = 'icons/mob/in-hand/right/guns_experimental.dmi')
 	recoil = 1
-	flags = FPRINT | TWOHANDABLE
-	w_class = 5.0//we be fuckin huge maaan
-	fire_delay = 0
-	var/max_shells = 200
+	flags = FPRINT
+	w_class = 3.0
+	fire_delay = 2
+	fire_sound = 'sound/weapons/grenadelauncher.ogg'
+	var/list/loaded = list()
+	var/list/fired = list()
+
 	var/current_shells = 200
 
+/obj/item/weapon/gun/stickybomb/New()
+	..()
+	loaded = list(
+		new /obj/item/stickybomb(src),
+		new /obj/item/stickybomb(src),
+		new /obj/item/stickybomb(src),
+		new /obj/item/stickybomb(src),
+		new /obj/item/stickybomb(src),
+		new /obj/item/stickybomb(src),
+		)
+
+/obj/item/weapon/gun/stickybomb/Destroy()
```

Comment:
The sticky is placed in the chamber right as you fire it, in process_chambered().
Since you cannot remove stickies from a launcher without firing them it makes no real difference whether there's a sticky loaded in the chamber or not before you fire it.